pkg/p2p/libp2p/internal/handshake/handshake.go

```diff
@@ -24,10 +24,15 @@ import (
 )
 
 const (
-	ProtocolName    = "handshake"
+	// ProtocolName is the text of the name of the handshake protocol.
+	ProtocolName = "handshake"
+	// ProtocolVersion is the current handshake protocol version.
 	ProtocolVersion = "1.0.0"
-	StreamName      = "handshake"
-	messageTimeout  = 5 * time.Second // maximum allowed time for a message to be read or written.
+	// StreamName is the name of the stream used for handshake purposes.
+	StreamName = "handshake"
+	// MaxWelcomeMessageLength is maximum number of characters allowed in the welcome message.
+	MaxWelcomeMessageLength = 140
+	messageTimeout          = 5 * time.Second
```

Comment:
I know it's not in your PR, but this `const` is not used anywhere. Can we remove it?